net-analyzer/prelude-correlator/files/prelude-correlator-4.1.1-fix_bruteforce_rule.patch

```diff
@@ -0,0 +1,11 @@
+--- a/rules/BruteForcePlugin.py
```

Comment:
Are those upstream patches? Please include link to upstream VCS / some indication where they come from.